src/current/v23.1/molt-verify.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Use the MOLT Verify tool
+summary: Learn how to use the MOLT Verify tool to check for post-migration data discrepancies.
+toc: true
+docs_area: migrate
+---
+
+The MOLT Verify tool checks for post-migration data discrepancies between a source and a target database.
+
+The tool performs the following verifications to ensure post-migration data integrity:
+
+- **Table Verification:** Check that the structure of tables between the source database and the target database are the same.
+- **Column Definition Verification:** Check that the column names, data types, constraints, nullability, and other attributes between the source database and the target database are the same. 
+- **Row Value Verification:** Check that the actual data in the tables are the same between the source database and the target database.
+
+## Supported databases
+
+The following databases are currently supported:
+
+- PostgreSQL
+- MySQL
+
+## Use the MOLT Verify tool
+
+1. Get the JDBC connection strings for the source and target databases you want to compare.
+2. Make sure the SQL user running the MOLT verify tool has access to the necessary tables.
+3. Install and build the tool: 
+
+    Run the following commands on your local machine:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    git clone https://github.com/cockroachdb/molt.git
+    ~~~
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    cd molt
+    ~~~
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    go build .
+    ~~~
+    
+    To cross-compile, use:
+    
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    GOOS=linux GOARCH=amd64 go build -v .
+    ~~~ 
+    
+    Run the following command to verify the installation:
+	  
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    ./molt verify --help
+    ~~~
+
+4. Run MOLT Verify: 
+
+    The `molt verify` command takes two or more JDBC connection strings as arguments. You can append a name for easier readability using `<name>===` in front of the JDBC string. The first argument is considered as the "source of truth". 
+    
+    Examples:
+
+    The following example compares a PostgreSQL database with a CockroachDB database (simplified naming for both instances):
```

Comment:
the "simplified naming" line is confusing. do you mean that the connection strings contain placeholders?

Reply:
it's actually unnecessary now that I think of it. Deleting it